tarantool/connection.py

```diff
@@ -1368,7 +1446,7 @@ def ping(self, notime=False):
             return "Success"
         return t1 - t0
 
-    def select(self, space_name, key=None, *, offset=0, limit=0xffffffff, index=0, iterator=None):
+    def select(self, space_name, key=None, *, offset=0, limit=0xffffffff, index=0, iterator=None, **kwargs):
```

Comment:
Is there any examples of receiving push messages in select/replace/etc?

Reply:
I added an example with `insert` via `tarantool.ConnectionPool` to the PR description. I plan to transfer the description from the current PR to `/docs/source/quick-start.rst`